configure.ac

```diff
@@ -470,10 +470,16 @@ AM_CONDITIONAL([ENABLE_DOCS], [test "x$enable_docs" = "xyes"])
 if test "x$enable_docs" = "xyes"; then
 	doxygen_version=$($DOXYGEN --version)
 	AS_VERSION_COMPARE([$doxygen_version], [1.8.11],
-					   [],
-					   [],
-					   [AC_MSG_ERROR([Doxygen $doxygen_version too new: versions higher than 1.8.11 are known to render poorly.])]
-					  )
+                           [],
+                           [],
+                           [
+                            AS_VERSION_COMPARE([$doxygen_version], [1.8.14],
+                                               [AC_MSG_ERROR([Doxygen $doxygen_version not usable: versions between 1.8.12 and 1.8.14 are known to render poorly.])],
```

Comment:
Won't this only appear for version < 1.8.14, and not when it's exactly 1.8.14? If I got the check right, this means that for 1.8.14 this will be considered a success, go on, and since 1.8.14 is affected too, cause broken docs. If so, the check should probably be on 1.8.15? (just as we have 11 instead of 12 in the previous one).

Reply:
Sorry, I did not explain the situation well. The flag to opt out of the new style menu was released in 1.8.14, so 1.8.14 works.

Reply:
Oh, perfect then, thanks! I was just testing this with the 1.8.15 on Fedora and it works perfectly, so I'll merge :+1: